### PR TITLE
Added works and aboutme

### PR DESCRIPTION
**Pull request by LeslieAine**
In this Section I;
1. Added a Works page and About me page
2. Added the styles to the pages using css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,4 +45,91 @@ <h2 class="describe">I am a Software Developer</h2>
...
+
+   <ul class="language">
+       <a class="icon" href="#"><img src="images/skills.png" alt=""/></a>
+       <p> Skills </p>
+   </ul>
+   <ul class="list">   
+       <a class="skillset" href="#">Codekit</a>
+       <a class="skillset" href="#">Github</a>
+       <a class="skillset" href="#">Codepen</a>
+       <a class="skillset" href="#">Gitlab</a>
```
Comment (Olamarx) [inline index.html:127]: - [ ]  In this section, The technology categories ("Languages", "Frameworks" and "Skills") should be in a ul, each of them in a li tag and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. This is to make the listing orderly and semantically correct.

- [ ] Kindly remeber to align the content properly

### Your design
image

### Figma design
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @LeslieAine,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- All linters passed ✅
- Commit messages ✔️ 
- Reame file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -90,3 +90,224 @@ ul {
...
+  height: 0;
+  margin-bottom: 60px;
+}
+
+.bgimage {
+  background-image: url(images/works.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  border: inset 170px transparent;
+  box-sizing: border-box;
```
Comment (GedeonTS) [inline styles.css:124]: - [ OPTIONAL]  Good job adding the section requested by the previous reviewer :+1: , Kindly stick to the design as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using), regarding the background size and the font :recycle: 

  - Your design:
yourdesign

  - Figma design:
figmad

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @LeslieAine ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 

- Linter checks are green :heavy_check_mark: 
- Good use of github flow :+1: 

## Optional suggestions :small_airplane: 

__Kindly check the comments under the review__
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**